TSOClient/tso.simantics/NetPlay/EODs/Handlers/VMEODHoldEmCasinoPlugin.cs

```diff
@@ -64,13 +65,22 @@ public VMEODHoldEmCasinoPlugin(VMEODServer server) : base(server)
 
             DealerIntermissionComplete = true;
             DealerEventsQueue = new List<VMEODEvent>();
+            SyncQueue = new List<VMEODClient>();
         }
         #region Public
         public override void Tick()
         {
             if (Controller == null)
                 return;
 
+            // sync any latecomers' UIEODs
+            if (SyncQueue.Count > 0)
```

Comment:
Wrap this in `lock (SyncQueue) {`, and use `SyncQueue.Clear()` (need the reference to not change for locking reasons)